sass/style.sass

```diff
@@ -8,3 +7,6 @@
 @import "_tools.sass"
 @import "_components.sass"
 @import "_trumps.sass"
+
+//Page Specific Sass
+@import "_camper_blogs.scss"
```

Comment:
Trumps are you highest specificity elements on the site. They should be placed at the end of your CSS to override the styles that came before them.